### DISCUSSION TITLE
List live channel title/game in tooltip when hovering over tab

### DISCUSSION THREAD

**Discussion by romyDank** [Chatterino/chatterino2#3695] [Ideas]
When you hover over a tab you have in chatterino, after you see the red dot that they are live, you can see their title/game category so you can see if you want to watch them through chatterino instead of having to go to the webpage. 

![image](https://user-images.githubusercontent.com/104437689/165302812-291642a7-f2d0-4a47-8586-22436f49f7be.png)

[felanbird]: here's another visual example for it
![c2_3695](https://user-images.githubusercontent.com/41973452/165305569-dc6614b4-003a-4d27-9c3e-c8dc74519655.png)

**Comment by Felanbird** [comment]
This should probably respect existing `Chat title` settings
